h/static/scripts/annotator/anchoring/test/html-test.js

```diff
@@ -191,4 +200,50 @@ describe('HTML anchoring', function () {
     });
     return Promise.all(anchored);
   }, testCases);
+
+  describe('Web page baselines', function () {
+    var fixtures = require('./html-baselines');
+    var frame;
+
+    before(function () {
+      frame = document.createElement('iframe');
+      document.body.appendChild(frame);
+    });
+
+    after(function () {
+      frame.remove();
+    });
+
+    unroll('generates selectors which match the baseline (#name)', function (fixture) {
+      var fixtureHtml = fixture.html;
+      var annotations = fixture.annotations.rows;
+
+      // Break references to external stylesheets and images so that the test
+      // runner does not try to load them.
+      //
+      // This method is crude but Good Enough™ for our test fixtures and does
+      // not modify the offset of any substrings within the text content of the
+      // document for matches that occur in a text-content context (eg.
+      // <pre>, <noscript>).
+      fixtureHtml = fixtureHtml.replace(/src=/g, 'sr_=').replace(/href=/g, 'hre_=');
```

Comment:
Could do. This just makes it a little easier to ensure consistent treatment of markup across each test case.